spec/support/schemas/rated_disability_base.json

```diff
@@ -3,7 +3,7 @@
   "required": [
     "decision_code",
     "decision_text",
-		"diagnostic_code",
+    "diagnostic_code",
```

Comment:
this file was previously using tabs on this line (which consequently broke my camelizing script), so i switched it to spaces